### PR TITLE
simplify the readme

### PR DESCRIPTION
**Pull request by bruno-ds**
evict version related content

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -49,45 +49,17 @@ iTop also offers mass import tools and web services to integrate with your IT
 
 
 
-## Last releases
+## Lastest release
```
Comment (piRGoif) [inline README.md]: Shouldn't this be "Latest" instead of "Lastest" ?
```suggestion
## Latest release
```

**Review by piRGoif — CHANGES_REQUESTED** [review]
Perfect, thanks Bruno !

I would also move the "resources" section below "Latest release", so that TOC would change from :

> Features, Resources, Latest release, About us, Contributors

To :

> Features, Latest release, Resources, About us, Contributors

**Comment by bruno-ds** [comment]
> Perfect, thanks Bruno !
> 
> I would also move the "resources" section below "Latest release", so that TOC would change from :
> 
> > Features, Resources, Latest release, About us, Contributors
> 
> To :
> 
> > Features, Latest release, Resources, About us, Contributors

IMO, this file lust be easier for the newcomers, and a newcomer is more interested in finding resources than infos on the latest release (which interest more experienced users).

**Comment by bruno-ds** [comment]
@v-dumas : your feedback concerning replacement of the versioned link would be appreciated:
[63]: https://www.itophub.io/wiki/page?id=latest:release:2_7_whats_new

(we are waiting for an orphaned page to become more visible)

**Comment by bruno-ds** [comment]
> Perfect, thanks Bruno !
> 
> I would also move the "resources" section below "Latest release", so that TOC would change from :
> 
> > Features, Resources, Latest release, About us, Contributors
> 
> To :
> 
> > Features, Latest release, Resources, About us, Contributors

This is done, could you please review and if it please you: squash + merge?

**Review by Molkobain — APPROVED** [review]

**Comment by Molkobain** [comment]
Looks all good to me!

**Review by piRGoif — APPROVED** [review]

**Comment by Molkobain** [comment]
@bruno-ds we can't merge it, there are conflicts apparently. can you check it out?

**Comment by bruno-ds** [comment]
> @bruno-ds we can't merge it, there are conflicts apparently. can you check it out?

yes, this commit generate a conflict: 
https://github.com/Combodo/iTop/commit/70835984de093d572eeb9efd0cdbe6b40b837431

I'll look at it :)

**Pull request merged**